### PR TITLE
Dev removepy2

### PR DESCRIPTION
**Pull request by bardkw**
The objective of this PR is to remove six as a dependency

### CODE REVIEW

**Comment by Purg** [comment]
This looks like it could possibly use a rebase instead of a master-merge (there's more complexity added with the merge than the rebase).

**Comment by Purg** [comment]
Is it correct that the "removed instances of six" commit is the only new one being added here? It seems that there are two other commits being duplicated (incorrectly) here.

**Comment by bardkw** [comment]
think i'm done showing off my git-illiteracy here. should be good now

File: tests/representation/DataSet/test_kvstore_backed.py [hunk 1]
```diff
@@ -80,8 +78,8 @@ def test_add_data(self) -> None:
         mem_kv = MemoryKeyValueStore()
         kvds = KVSDataSet(mem_kv)
 
-        de1 = DataMemoryElement(six.b('bytes1'))
-        de2 = DataMemoryElement(six.b('bytes2'))
+        de1 = DataMemoryElement(bytes('bytes1', 'utf-8'))
```
Comment (Purg) [inline tests/representation/DataSet/test_kvstore_backed.py:81]: Could this just be `b'bytes1'`? I don't know if I'm missing something where UTF-8 encoding is required or preferred.

Comment (bardkw) [reply tests/representation/DataSet/test_kvstore_backed.py]: updated to use b'bytes1'

**Review by Purg — APPROVED** [review]

**Pull request merged**